packages/create-sitecore-jss/src/common/processes/transform.ts

```diff
@@ -192,6 +192,25 @@ export const diffAndWriteFiles = async ({
   }
 };
 
+export const populateEjsData = (answers: BaseArgs, destination?: string) => {
+  const ejsData: Data = {
+    ...answers,
+    helper: {
+      isDev: isDevEnvironment(destination || answers.destination),
+      getPascalCaseName: getPascalCaseName,
+      getAppPrefix: getAppPrefix,
+    },
+  };
+  // calculate relative path between main app and proxy when needed
```

Comment:
Add a bit more details in comments, why it's added and when it's used
```suggestion
  // When SPA application and XM Cloud proxy is used we need to calculate relative path between SPA app and proxy to apply that in EJS template. Absolute path can't be used across all the machines
```

Reply:
For this and comments below, it still can apply to non-angular app. I'll keep it as is